Converters/CSi/Speckle.Converters.CSiShared/ServiceRegistration.cs

```diff
@@ -14,12 +17,20 @@ public static IServiceCollection AddCSiConverters(this IServiceCollection servic
   {
     var converterAssembly = Assembly.GetExecutingAssembly();
 
+    // Register top-level converters
     serviceCollection.AddTransient<CSiObjectToSpeckleConverter>();
+    serviceCollection.AddRootCommon<CSiRootToSpeckleConverter>(converterAssembly);
+
+    // Register extractors
     serviceCollection.AddScoped<DisplayValueExtractor>();
+    serviceCollection.AddScoped<ClassPropertyExtractor>();
 
-    // TODO: Property extractor
+    // Register geometry converters
+    serviceCollection.AddTransient<ITypedConverter<CSiFrameWrapper, Line>, LineToSpeckleConverter>();
```

Comment:
We do not need to register `ITypedConverter`s since they are handled magically

Reply:
Gone :)